controllers/add_pods.go

```diff
@@ -38,13 +38,14 @@ type AddPods struct{}
 // Reconcile runs the reconciler's work.
 func (a AddPods) Reconcile(r *FoundationDBClusterReconciler, context ctx.Context, cluster *fdbtypes.FoundationDBCluster) *Requeue {
 	configMap, err := internal.GetConfigMap(cluster)
+	logger := log.WithValues("cluster", cluster.Name, "reconciler", "AddPods")
```

Comment:
Can we add `"namespace", cluster.Namespace,` that will always be the same like the cluster.

Reply:
done